### PR TITLE
Add-DbaComputerCertificate, CheckList Changes

### PR DESCRIPTION
**Pull request by gbargsley**

### CODE REVIEW

File: functions/Add-DbaComputerCertificate.ps1 [hunk 1]
```diff
@@ -54,87 +54,87 @@ Add-DbaComputerCertificate -Path C:\temp\cert.cer
...
-			}
-		}
-	}
+      if ($PScmdlet.ShouldProcess("local", "Connecting to $computer to import cert")) {
+        try {
+          Invoke-Command2 -ComputerName $computer -Credential $Credential -ArgumentList $certdata, $Password, $Store, $Folder -ScriptBlock $scriptblock -ErrorAction Stop |
+            Select-DefaultView -Property FriendlyName, DnsNameList, Thumbprint, NotBefore, NotAfter, Subject, Issuer
+        }
+        catch {
+          Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $computer -Continue
```
Comment (potatoqualitee) [inline functions/Add-DbaComputerCertificate.ps1]: Hey @gbargsley - the connection errors are just for the top connecting part. When in doubt, you can leave that part off.

File: functions/Add-DbaComputerCertificate.ps1 [hunk 2]
```diff
@@ -54,87 +54,87 @@ Add-DbaComputerCertificate -Path C:\temp\cert.cer
...
+        Write-Verbose "Importing cert to $Folder\$Store"
+        $tempStore = New-Object System.Security.Cryptography.X509Certificates.X509Store($Folder, $Store)
+        $tempStore.Open('ReadWrite')
+        $tempStore.Add($cert)
+        $tempStore.Close()
 				
-				Write-Verbose "Searching Cert:\$Store\$Folder"
-				Get-ChildItem "Cert:\$store\$folder" -Recurse | Where-Object { $_.Thumbprint -eq $cert.Thumbprint }
-			}
+        Write-Verbose "Searching Cert:\$Store\$Folder"
```
Comment (potatoqualitee) [inline functions/Add-DbaComputerCertificate.ps1]: Thank you for keeping this as Write-Verbose! I don't know if it's in @wsmelton's notes, but when something is in a script block and thus will be executed remotely, it has to use native commands /cc @jpomfret @csolje

Comment (gbargsley) [inline functions/Add-DbaComputerCertificate.ps1]: @potatoqualitee  Not sure if I should approve since I submitted.

Comment (potatoqualitee) [inline functions/Add-DbaComputerCertificate.ps1]: Attached screenshots of working examples will do for those 👍

**Comment by gbargsley** [comment]
@potatoqualitee @niphlod 
Do we want a pop up?

Other tests worked.

image

image

**Comment by potatoqualitee** [comment]
yeahhh bawww that's good. prompt is fine.

**Pull request merged**